src/content/self-hosted/install/releases.mdx

```diff
@@ -6,11 +6,26 @@ id: releases
 ---
 
 ## 1.11.0
-TBD
+2 August 2023
+
+### Features
+
+- Upgrade buildkit v0.11.6. <!-- 6771 -->
+- Upgrade ingress-nginx to 4.7.1. <!-- 6838 -->
 
 ### Improvements
 
-- Added [configuration setting in the daemonset](self-hosted/administration/configuration.mdx#daemonset) to disable the configuration of private registry credentials in the nodes <!-- Ticket: 6741, PR: 6742 -->
+- Underlaying Kubernetes cluster version is now shown in the Admin section. <!-- 6734 -->
+- A warning message is now shown when GitHub App private key is not valid. <!-- 6750 -->
```

Comment:
@jpf-okteto where do we show this, in the logs?

Reply:
In the logs but we also display a more meaningful error in the UI, in the red banner